src/components/pages/Alerts/RenderAlerts.js

```diff
@@ -95,9 +80,13 @@ function RenderAlerts(props) {
                             <h4>Time: {alert.createddate}</h4>
                           </div>
                           <div className="buttDiv">
-                            <Button type="primary" /*onClick={showModal}*/>
+                            <Button
+                              type="primary"
+                              onClick={() => resolveBtn(alert.id)}
+                            >
                               Resolve
                             </Button>
+
                             {/* <Modal
```

Comment:
We can remove this commented out code later, if it is not used

Reply:
Yes I agree, I saw a possible use for this if we added an add Clubs functionality, so I figured that leaving it would be a safe option.

Reply:
I think this can be removed as well. You can copy/paste it to a gist or something and it's also stored in a commit that you can reference back to later in the case where you'd potentially be using it.